packages/malloy/src/malloy.ts

```diff
@@ -380,7 +383,8 @@ export class Malloy {
 
   static compileSQLBlock(
     partialModel: ModelDef | undefined,
-    toCompile: SQLBlockSource
+    toCompile: SQLBlockSource,
+    options?: {replaceMaterializedReferences?: boolean}
```

Comment:
Same reason, a SQL block might contain queries that need to have stages materialized.